vncviewer/Viewport.cxx

```diff
@@ -606,6 +606,11 @@ int Viewport::handle(int event)
     if (Fl::event_button3())
       buttonMask |= 4;
 
+    if(Fl::event_buttons()&FL_BUTTON(4))
+        buttonMask |= 128;
+    if(Fl::event_buttons()&FL_BUTTON(5))
+        buttonMask |= 256;
```

Comment:
Have you looked at Windows and macOS support? It seems FLTK doesn't natively support those, but we might be able to fetch what we need indirectly.

Reply:
I've been testing from WSL and an Arch laptop i have. I need to set up my build chain for windows. I don't have a mac though.